server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/SieveMailet.java

```diff
@@ -20,14 +20,12 @@
 
```

Comment:
is the class already covered by tests ? I see code changes but I didn't saw tests for now ...